src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -351,6 +351,46 @@ export default class ClinicalData extends React.Component<IClinicalDataProps, {}
         }
     }
 
+    @computed get boxPlotTooltip() {
+        return (d: any) => {
+            let content;
+            if (this.boxPlotData.isComplete) {
```

Comment:
is it even possible for the tooltip to show even the boxplot data hasn't completed?

Reply:
no